### PR TITLE
Upstream merge up to November

### PR DESCRIPTION
**Pull request by carterqw2**
### Description
Merges upstream commits up to ebc9acf8013dd1a2c05c6605953482e2be8bd250

### Issues
Relates to https://github.com/celo-org/data-services/issues/557

### CODE REVIEW

File: apps/indexer/lib/indexer/block/fetcher.ex [hunk 1]
```diff
@@ -84,7 +84,7 @@ defmodule Indexer.Block.Fetcher do
   # These are all the *default* values for options.
   # DO NOT use them directly in the code.  Get options from `state`.
 
-  @receipts_batch_size 250
+  @receipts_batch_size 100
```
Comment (shazarre) [inline apps/indexer/lib/indexer/block/fetcher.ex:87]: @carterqw2 @rkachowski how do we feel about it?

File: apps/indexer/lib/indexer/fetcher/coin_balance.ex [hunk 2]
```diff
@@ -22,7 +22,7 @@ defmodule Indexer.Fetcher.CoinBalance do
 
   @defaults [
     flush_interval: :timer.seconds(3),
-    max_batch_size: 500,
+    max_batch_size: 100,
```
Comment (shazarre) [inline apps/indexer/lib/indexer/fetcher/coin_balance.ex:25]: and this?

File: docker-compose/envs/common-blockscout.env [hunk 3]
```diff
@@ -79,6 +79,7 @@ DISABLE_READ_API=false
 DISABLE_WRITE_API=false
 DISABLE_INDEXER=false
 DISABLE_REALTIME_INDEXER=false
+DISABLE_TOKEN_INSTANCE_FETCHER=false
```
Comment (shazarre) [inline docker-compose/envs/common-blockscout.env:82]: we need a corresponding infra PR to make things explicit or we rely on the default value?

Comment (carterqw2) [reply docker-compose/envs/common-blockscout.env:82]: I would rely on default values here.

File: apps/indexer/lib/indexer/block/fetcher.ex (hunk 1 above)
Comment (carterqw2) [reply apps/indexer/lib/indexer/block/fetcher.ex:87]: Looks like smaller batch sizes is a way to go in general.

Comment (shazarre) [reply apps/indexer/lib/indexer/block/fetcher.ex:87]: agree with both of you, I haven't noticed that receipts are a bottleneck, although reducing a single batch size is (usually) a good idea, worth trying out if this affects us anyhow and also knowing why it's being reduced here

**Review by rkachowski — APPROVED** [review]

**Pull request merged**